app/validators/form_password_validator.rb

```diff
@@ -40,11 +32,19 @@ def password_graphemes_length
     end
   end
 
-  def password_score
-    @password_score = ZXCVBN_TESTER.test(
-      password,
-      user.email_addresses.flat_map { |address| ForbiddenPasswords.new(address.email).call },
-    )
+  def password_email
+    @emails_strings = user.email_addresses.
+      flat_map { |address| ForbiddenPasswords.new(address.email).call }
+    @emails = false
+    @emails_strings.each do |string|
+      if password.include? string
```

Comment:
The previous behavior only flagged a password if it was exactly equal to a forbidden password. I don't necessarily think that was the _right_ way to approach it, but it'd be worth having discussion if we want to change that behavior as part of this ticket, since it wasn't originally in scope.

Reply:
I figured that was what was happening here:

```suggestion
      def password_score
    @password_score = ZXCVBN_TESTER.test(
      password,
      user.email_addresses.flat_map { |address| ForbiddenPasswords.new(address.email).call },
    )
  end
```
We were previously appending optional values to zxcvbn consisting of the user's email and also Login.gov. This change will display an error if the user tries using their email (or significant parts of it) as a password.